mlflow/utils/model_utils.py

```diff
@@ -162,6 +163,20 @@ def _validate_and_copy_code_paths(code_paths, path, default_subpath="code"):
     return code_dir_subpath
 
 
+def _validate_and_copy_model_code_path(code_path, path, default_subpath="model_code"):
```

Comment:
default_subpath= FLAVOR_CONFIG_MODEL_CODE
Can we update the above so we don't define this var in 2 places?

Reply:
yes, this also might be removed in this pr https://github.com/mlflow/mlflow/pull/11843/files or the previous 🤔